packages/react-next/src/components/Link/Link.types.ts

```diff
@@ -72,7 +72,9 @@ export interface ILinkProps extends ILinkHTMLAttributes<HTMLAnchorElement | HTML
   theme?: ITheme;
 
   /**
-   * Optional keytip for this Link
+   * Optional keytip.
+   *
+   * @deprecated This no longer works. Use `useKeytipData` hook instead.
```

Comment:
This doesn't seem like the right deprecation message - is there any way to have this value deprecated in v7 with an alternative implementation for customers?

Reply:
If we can deprecate it (but keep it functional)_ with the viable alternative in v7, and then remove it in v8, that's a better user experience. They can update to wrapping things before the upgrade, right now, in v7.

Reply:
Unfortunately user can't use the new approach on V7 components, because they don't support passing `ref` to the root element